azure-iot-device/azure/iot/device/provisioning/abstract_provisioning_device_client.py

```diff
@@ -104,13 +108,6 @@ def register(self):
         """
         pass
 
-    @abc.abstractmethod
-    def cancel(self):
```

Comment:
> def cancel(self): [](start = 3, length = 18)

Removing cancel is something that should be done before GA since it's a public interface change.